### PR TITLE
Review of s/mime documentation and minor changes proposed

### PR DESCRIPTION
**Pull request by qosobrin**
I reviewed the text. I think it is ready to be published, although there might be some parts that could be improved. Maybe in the next round it could be completed, but I think it is not worth the extra effort right now. 

Thank you for the effort. Regards.

### CODE REVIEW

File: manual/customize/smime.md [hunk 1]
```diff
@@ -23,48 +23,47 @@ Requirements
     Note:
 
       * On Sympa prior to 6.2, openssl(1) utility was required.  It is no
-        longer required, however, to install Perl modules above, openssl
-        library is required.
+        longer required but it can be a prerrequisite to install the Perl
```
Comment (racke) [inline manual/customize/smime.md]: Typo: it is "prerequisite" so you have a "r" too much.

File: manual/customize/smime.md [hunk 2]
```diff
@@ -152,7 +151,7 @@ Note:
     be escaped to avoid limitation of filesystem encoding.
     By historical reason, escaping scheme is slightly wierd (`escape_chars()`
     in [Sympa::Tools::Text](../man/Sympa-Tools-Text.3.md) is used).
-    This will be fixed on the future release of Sympa.
+    This will be fixed on a future release of Sympa.
```
Comment (racke) [inline manual/customize/smime.md]: Shouldn't it be written "in a future release"?

File: manual/customize/smime.md [hunk 3]
```diff
@@ -208,10 +208,10 @@ Once certificate is obtained by Sympa, message encryption is available:
...
      If decryption fails, encrypted message is delivered intact.
 
-  3. At last when Sympa delivers message, it tries to encrypt message again
-     using certificate of recipient by each, then delivers it.
+  3. Finally, when Sympa delivers the message, it tries to encrypt it again
+     using each recipient's certificate and delivers it.
 
-     If encryption fails (e.g. certificate of the recipient is not found),
+     If encryption fails (e.g. recipient's certificate is not found),
      Sympa delivers a message informing failure instead (a mail template
```
Comment (racke) [inline manual/customize/smime.md]: "informing about the failure" sounds better to me.

File: manual/customize/smime.md (hunk 1 above)
Comment (qosobrin) [reply manual/customize/smime.md]: You are right, spanish influence. Corrected.

**Comment by racke** [comment]
OK, I would like to merge this PR unless someone has objections.

**Comment by qosobrin** [comment]
Are there any other necessary steps for me to do in order to get these changes merged? Am I missing something?

**Comment by racke** [comment]
The reminder was the missing step :stuck_out_tongue_winking_eye:. Sorry for the delay. @qosobrin Thanks a lot for your contribution.

**Comment by ikedas** [comment]
Good! Thanks @qosobrin. You are an [author](https://github.com/sympa-community/sympa-community.github.io/blob/a49689a/AUTHORS.md).

**Pull request merged**